supervisor_types.go

```diff
@@ -116,18 +116,32 @@ type Transform struct {
 	Expr string `json:"expression"`
 }
 
+// SpatialDimension represents single spatial dimension datum.
+// https://druid.apache.org/docs/latest/querying/geo/#spatial-indexing
+type SpatialDimension struct {
+	DimName     string   `json:"dimName"`
```

Comment:
nit: As we already used longer version for the filed below, can we also have this filed more verbose? (DimensionsName)

That or use `Dims` to be consistent with the spec.

WDYT? What is being used elsewhere?

Reply:
Accepted. 👍 I will change field name to `Dims` here.